dojo/settings/settings.dist.py

```diff
@@ -1568,6 +1571,7 @@ def saml2_attrib_map_format(dict):
     'PYSEC': 'https://osv.dev/vulnerability/',
     'SNYK': 'https://snyk.io/vuln/',
     'RUSTSEC': 'https://rustsec.org/advisories/',
+    'VNS': 'https://vulners.com/',
```

Comment:
```suggestion
```
Not needed, see my other comments

Reply:
Sorry, didn't find other comments regarding this feature. I used this to highlight links that follows to CVE and other advisories located at vulners.com.